tensorflow/compiler/xla/service/gpu/tests/gpu_atomic_test.cc

```diff
@@ -104,7 +104,7 @@ TEST_F(GpuAtomicTest, TestAddAtomicF32) {
 )";
 
   CompileAndVerifyIr(hlo_string, is_built_with_rocm_ ? R"(
-CHECK: atomicrmw fadd float addrspace{{.*}}, float {{.*}} seq_cst, align 4
```

Comment:
This change could be split off

Reply:
Split it into it's own PR https://github.com/tensorflow/tensorflow/pull/58202